### PR TITLE
feat(solidjs-pod): Fix starter kit greeting component

### PR DESCRIPTION
**Pull request by SDaian**
## Type of change

- [ ] Documentation change
- [x] Bug fix

## Summary of change

- [x] there is an issue with the network call, which needs to be updated for Netlify
- [x] the input field needs to go away, it's not necessary. See SvelteKit as an example, or the NextJS kit - both examples of how this should work

Closes: #823

### CODE REVIEW

**Review by Megio — APPROVED** [review]
LGTM 🛸

File: starters/solidjs-tailwind/src/components/FetchExample/Greeting.spec.jsx [hunk 1]
```diff
@@ -1,22 +1,52 @@
...
   it('should mount', async () => {
-    const wrapper = await render(() => <Greeting />);
     expect(wrapper).toBeTruthy();
   });
 
   it('should show the mocked greeting', async () => {
-    await render(() => <Greeting />);
-    const text = await screen.findByText('Message: Hi Learner');
-    expect(text).toBeVisible();
+    expect(wrapper.findByText('This Dot Labs!')).toBeTruthy();
```
Comment (hdJerry) [inline starters/solidjs-tailwind/src/components/FetchExample/Greeting.spec.jsx:50]: This test isn't valid as whatever test you put still passes.

eg
```js
 expect(wrapper.findByText('nothing')).toBeTruthy();
```

File: starters/solidjs-tailwind/src/components/FetchExample/Greeting.spec.jsx [hunk 2]
```diff
@@ -1,22 +1,52 @@
...
 describe('Greeting', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = render(() => (
+      <Router>
+        <Greeting />
+      </Router>
+    ));
+
+    window.fetch = () => {
```
Comment (hdJerry) [inline starters/solidjs-tailwind/src/components/FetchExample/Greeting.spec.jsx:29]: Not sure the purpose of this as its absence doesn't affect the result

File: starters/solidjs-tailwind/src/components/FetchExample/Greeting.spec.jsx (hunk 1 above)
Comment (hdJerry) [reply starters/solidjs-tailwind/src/components/FetchExample/Greeting.spec.jsx:50]: this should be the test, you notice the await also.

```js
expect(await wrapper.findByText('Hi Learner from This Dot Labs!')).toBeTruthy();
```

As we are mocking the API, you can check the mock folder to see what we mocked

**Review by hdJerry — CHANGES_REQUESTED** [review]
Please, check my comments

File: starters/solidjs-tailwind/src/components/FetchExample/Greeting.jsx [hunk 3]
```diff
@@ -1,32 +1,16 @@
 import { createEffect, createResource, createSignal } from 'solid-js';
```
Comment (hdJerry) [inline starters/solidjs-tailwind/src/components/FetchExample/Greeting.jsx:1]: Please remove unused imports

**Review by hdJerry — CHANGES_REQUESTED** [review]

**Review by Megio — APPROVED** [review]
LGTM

**Pull request merged**